internal/campaigns/executor.go

```diff
@@ -422,11 +458,11 @@ func createChangesetSpecs(task *Task, completeDiff string, features featureFlags
 				BaseRev:        task.Repository.Rev(),
 				HeadRepository: task.Repository.ID,
 				HeadRef:        "refs/heads/" + branch,
-				Title:          task.Template.Title,
-				Body:           task.Template.Body,
+				Title:          title,
+				Body:           body,
 				Commits: []GitCommitDescription{
 					{
-						Message:     task.Template.Commit.Message,
+						Message:     message,
 						AuthorName:  authorName,
 						AuthorEmail: authorEmail,
```

Comment:
Should we make all the fields eligible for templates for consistency?

Reply:
Yeah, good idea. Just did that in 585abbd